result/dhcpv4client.yaml

```diff
@@ -0,0 +1,111 @@
+openapi: 3.0.3
+info:
+  title: DHCPv4 client results models
+  version: ^0.0.0
+components:
+  schemas:
+    Dhcpv4Client.Metrics.Request:
+      description: >-
+        The request to retrieve DHCPv4 per client metrics/statistics.
+      type: object
+      properties:
+        client_names:
+          description: >-
+            The names of DHCPv4 clients to return results for.
+            An empty list will return results for all DHCPv4 client.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Dhcpv4client/properties/name"
+          x-field-uid: 1
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned except for
+            any result_groups. The name of the DHCPv4 client cannot be excluded.
+          type: array
+          items:
+            type: string
+            x-enum:
+              session_state:
+                x-field-uid: 1
+              discovers_sent:
+                x-field-uid: 2
+              offers_received:
+                x-field-uid: 3
+              requests_sent:
+                x-field-uid: 4
+              acks_received:
+                x-field-uid: 5
+              nacks_received:
+                x-field-uid: 6
+              releases_sent:
+                x-field-uid: 7
+              declineds_sent:
```

Comment:
typo: declines_sent
We can also consider using "Tx"/"Rx" instead of sent/received.